deploy/roles/stata/defaults/main.yaml

```diff
@@ -1,9 +1,22 @@
 ---
-stata_install_directory: /usr/local/stata13
-stata_local_bin_directory: /artifacts
-stata_remote_bin_directory: /tmp
-stata_serial_number: STATA_SERIAL_NUMBER
-stata_code: STATE_CODE
-stata_authorization: STATA_AUTHORIZATION
-stata_line_one: Line 1
-stata_line_two: Line 2
+stata_install_directory:            /usr/local/stata13
+stata_local_bin_directory:          /artifacts
+stata_remote_bin_directory:         /tmp
+stata_serial_number:                STATA_SERIAL_NUMBER
+stata_code:                         STATE_CODE
+stata_authorization:                STATA_AUTHORIZATION
+stata_line_one:                     Line 1
+stata_line_two:                     Line 2
+
+stata_odbc_mgr:                     unixodbc
+stata_processor:                    4
+stata_site:                         "/usr/share/shared_ado"
+stata_profile_dir:                 /usr/local/stata14
```

Comment:
This is redundant with `stata_install_directory`, please remove this setting and use the other, but set it to `/usr/local/stata14`